src/main/java/alyssa/Ui.java

```diff
@@ -0,0 +1,43 @@
+package alyssa;
+
+/**
+ * This class helps Alyssa interact with the user.
+ */
+public class Ui {
+    private static final String LINE = "____________________________________________________________";
+    private static final String LOGO = " ___  __    __   __  ____   ____    ___\n"
+            + "|   | | |   \\ \\ / / |  __| |  __|  |   |\n"
+            + "|   | | |    \\   /   \\ \\    \\ \\    |   |\n"
+            + "|___| | |     | |     \\ \\    \\ \\   |___|\n"
+            + "|   | | |___  | |     _\\ \\   _\\ \\  |   |\n"
+            + "|   | |_____| |_|    |____| |____| |   |\n";
+
+    /**
+     * Sends a greeting message via System.out.println.
+     */
+    protected String getGreeting() {
+        String output = "Hello! I'm Alyssa!\n";
+        output += "What can I do for you?\n";
+        return output;
+    }
+    /**
+     * Returns a goodbye message.
+     */
+    protected String goodbye() {
```

Comment:
I understand that the intention is obvious, but since this is a method, perhaps it's more suitable to name it as `sayGoodbye` or `printGoodbye`?